template/package.json

```diff
@@ -55,10 +55,10 @@
     "jest": "^26.6.3",
     "jest-fetch-mock": "^3.0.3",
     "jest-transform-stub": "^2.0.0",
-    "metro-react-native-babel-preset": "^0.67.0",
+    "metro-react-native-babel-preset": "^0.72.1",
     "react-native-config-node": "^0.0.3",
     "react-native-svg-transformer": "^0.14.3",
-    "react-test-renderer": "17.0.2",
+    "react-test-renderer": "18.1.0",
     "redux-saga-test-plan": "^4.0.6",
     "typescript": "^3.8.3"
```

Comment:
This looks like a great opportunity to also tackle this issue #14 
This template was based on [react-native-template-typescript](https://github.com/react-native-community/react-native-template-typescript) and I see they currently have typescript set to ^4.8.3